src/display/Tag/Tag.spec.tsx

```diff
@@ -0,0 +1,60 @@
+import { fireEvent, render, screen } from "@testing-library/react";
+import React from "react";
+import { describe, expect, it, vi } from "vitest";
+import Tag from "src/display/Tag/Tag";
+import DeleteIcon from "src/icons/delete";
+
+describe("Tag Component", () => {
+  it("renders the base tag", () => {
+    render(<Tag>Tag</Tag>);
+    expect(screen.getByText("Tag")).toBeInTheDocument();
+  });
+
+  it("renders the Tag with a variant prop", () => {
```

Comment:
This test is the same the first one. I don't understand what is the différence

Reply:
modification based on Balkeo comment : 
https://github.com/Groupe-Gaillard/purrfect-ui/pull/35#discussion_r1671790684

Thus, this test won't be needed anymore